rules-topics/05_armor.md

```diff
@@ -104,28 +104,30 @@ While a player may wear armor wherever and however they choose as is thematicall
 >Example: Spurt the Kobold has purchased enough Armor Training to raise their total Armor Cap to 30 points.
-* They have not purchased any ranks in Fencing or Stalwart Defense, and so they must wear a phys rep of all their armor.
+* They have not purchased any ranks in Dexterity Armor, and so they must wear a phys rep of all their armor.
 * Spurt is wearing a chain shirt (16 points), hard leather greaves (6 points), hard leather pauldrons (6 points), and cloth bracers (2 points).
 * Spurt possesses an Armor Tag for 30 points of Armor.
 
 Altogether, Spurt is wearing a total of 30 Armor Points, and has an Armor Cap of 30, which means they are able to use all of their Armor Points during combat.
 
 >Example:  Villi the Forged has purchased enough Armor Training to raise their Armor Cap to 45 points.
-* They have purchased 2 ranks of Stalwart Defense, which allows them to have another 10 Armor Points without an armor phys rep.
-* They are wearing a plate torso piece (36 points), hard leather bracers (6 points) and hard leather greaves on their legs (6 points).
+* They are wearing a plate torso piece (32 points), and plate bracers (16 points).
 * Villi possesses an Armor Tag for 45 points of Armor.
 
-Villi is wearing a total of 48 Armor Points, plus their ranks in Stalwart Defense which brings their total to 58. Because this is over their Armor Cap, they will only be able to use 55 Armor Points.
+Villi is wearing a total of 48 Armor Points. Because this is over their Armor Cap, they will only be able to use 45 Armor Points.
 
 >Example:  Raphael the Elf has purchased enough Armor Training to raise their Armor Cap to 20 points.
-* They have purchased 4 ranks of Fencing, which allows them to have another 20 Armor Points without an armor phys rep.
-* They are wearing a chain torso piece (16 points) and leather bracers (4 points).
+* They have purchased 4 ranks of Dexterity Armor, which allows them to have 20 Armor Points without an armor phys rep or Armor Tag.
 * Raphael currently does not possess an armor tag.
 
-Raphael is wearing a total of 40 points of armor, 20 from phys reps and 20 from Fencing, however, because they do not possess an item tag for their armor, they may not benefit from any phys-repped armor. As such, their current armor total is 20 from Fencing. If Raphael gets an Armor Tag worth at least 20 points, they will be able to benefit from their full armor.
+Raphael is wearing a total of 20 points of armor from Dexterity Armor. 
+
+>Example:  Malokis the Tank has purchased enough Armor Training to raise their Armor Cap to 90 points.
+* They are wearing a plate torso piece (32 points), plate bracers (16 points) and plate greaves on their legs (16 points), and plate pauldrons on their shoulders (16 points).
+* Malokis possesses an Armor Tag for 80 points of Masterwork Armor. Which provides a bonus 10 armor points that do not need to be repped.
+
+Malokis is wearing a total of 90 points of armor from his plate and Masterwork Bonus. 
```

Comment:
Could you please also update the changelog file for your updates?

Reply:
Ah.  It will be similar to how you navigated back to your branch & this file for the update.  Instead, open up the changelog file and right at the top, add something like:

```
## v3.0.4 [2023-03-28]

---
### Armor
* Updated language, terms, and examples.
```

When I pull your branch into the release branch, it'll make it easier to consolidate and have a brief note on all the changes.  :D